test/common/io/io_uring_impl_test.cc

```diff
@@ -13,10 +14,11 @@ namespace {
 
 class TestRequest : public Request {
 public:
-  TestRequest(int& data) : data_(data) {}
+  TestRequest(int& data) : Request(RequestType::Read, mock_io_uring_socket_), data_(data) {}
```

Comment:
explicit ctor to avoid implicit conversions

Reply:
got it